### ISSUE TITLE
change default branch from master to main

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spreg#106]
We are way overdue in changing the default `spreg` branch from `master` to `main`.

xref https://github.com/pysal/mapclassify/issues/130

**Comment by jGaboardi** [comment]
@pedrovma I am going ahead to create a `main` branch and set that one as the default. We can leave the `master` branch up for now, just not as default. Then once there is a decision for #103, if it is merged into `master` then when can merge `master` into `main` and delete `master`. Sounds good?

**Comment by pedrovma** [comment]
Super. Thanks for taking care of this.
